docs/src/HQMTool.md

```diff
@@ -0,0 +1,512 @@
+		[Return:Int]     getSmoothingIterations(proj::Project)
+
+`status` is either "ON" or "OFF".
+
+To remove the smoother altogether,
+
+		[Return:nothing] removeSpringSmoother!(proj::Project)
+
+#### Manual Refinement
+
+Refinement can be specified either at a point, using the `RefinementCenter`, or along a line, using a `RefinementLine`. You can have as many of these as you want. They are useful if you know regions of the solution where refinement is needed (e.g. a wake) or in problematic areas in the geometry.
+
+To create a refinement center,
+
+		[Return:Dict{String,Any}] newRefinementCenter!(proj::Project, type::String,
+                          	 				           x0::Array{Float64}, h::Float64,
+                          	 				           w::Float64 )
+
+where the type is either `smooth` or `sharp`, `x0` = [x,y,z] is the location of the center, `h` is the mesh size, and `w` is the extent of the refinement region.
+
+Similarly, one can create a `RefinementLine`,
+
+		[Return:Dict{String,Any}] newRefinementLine!(proj::Project, type::String,
+                            				         x0::Array{Float64}, x1::Array{Float64},
+                           				             h::Float64,
+                                                     w::Float64 )
+
+where `x0` is the start postition and `x1` is the end of the line.
+
+To add a refinement region to the project,
+
+		[Return:nothing] addRefinementRegion!(proj::Project,r::Dict{String,Any})
+
+To get the indx'th refinement region from the project, or to get
+ a refinement region with a given name, use
+
+		[Return:Dict{String,Any}] getRefinementRegion(proj::Project, indx::Int)
+		[Return:Dict{String,Any}] getRefinementRegion(proj::Project, name::String)
+
+Finally, to get a list of all the refinement regions,
+
+		[Return:Array{Dict{String,Any}}] array = allRefinementRegions(proj::Project)
+
+A refinement region can be edited by using the following
+
+		[Return:nothing] 		 setRefinementType!(r::Dict{String,Any}, type::String)
+		[Return:String]  		 getRefinementType(r::Dict{String,Any})
+		[Return:nothing] 		 setRefinementLocation!(r::Dict{String,Any}, x::Array{Float64})
+		[Return:Array{Float64}]  getRefinementLocation(r::Dict{String,Any})
+		[Return:nothing] 		 setRefinementGridSize!(r::Dict{String,Any},h::Float64)
+		[Return:float64] 		 getRefinementGridSize(r::Dict{String,Any})
+		[Return:nothing] 		 setRefinementWidth!(r::Dict{String,Any},w::Float64)
+		[Return:float64] 		 getRefinementWidth(r::Dict{String,Any})
+
+where `r` is a dictionary returned by `newRefinementCenter!`, `newRefinementLine!`, or `getRefinementRegion`.
+
+To further edit a `RefinementLine`, use the methods
+
+		[Return:nothing] 		 setRefinementStart!(r::Dict{String,Any}, x::Array{Float64})
+		[Return:Array{Float64}]  getRefinementStart(r::Dict{String,Any})
+		[Return:nothing] 		 setRefinementEnd!(r::Dict{String,Any}, x::Array{Float64})
+		[Return:Array{Float64}]  getRefinementEnd(r::Dict{String,Any})
```

Comment:
Strange indentation (here and in other places)

Reply:
This should be fixed in the updated docs.